### DISCUSSION TITLE
Reverse Proxy?

### DISCUSSION THREAD

**Discussion by jsayles** [mozilla/hubs#2795] [General Questions & Discussion] [answered]
I'm looking into how to run Hubs Cloud behind a reverse proxy to obscure/expire the URLs.  I've tried a traditional reverse proxy setup and I'm not having any luck.  What more should I be looking for when setting this up?

**Comment by emclaren** [comment, marked as answer]
Hi @jsayles - apologies for the delay in responding, this was not appearing in our list of unanswered questions until just the other day.

Unfortunately this is a fairly advanced use-case and I don't have any helpful guidance. I was wondering if you were ever able to get this up and running, because if so I would love to hear how you went about this

**Comment by jsayles** [comment]
I have not found a solution yet.  I think I need to dive more into how the protocol works to understand how the routing needs to be set up.  Any hits or tips to point me in the right direction would be much appreciated.
